java-checks/src/main/java/org/sonar/java/checks/tests/AssertionArgumentOrderCheck.java

```diff
@@ -59,12 +65,17 @@
   protected MethodMatchers getMethodInvocationMatchers() {
     return MethodMatchers.or(
       MethodMatchers.create().ofTypes(ORG_JUNIT_ASSERT)
-        .names("assertEquals", "assertSame", "assertNotSame")
+        .names(ASSERT_EQUALS, ASSERT_SAME, ASSERT_NOT_SAME)
+        .withAnyParameters()
+        .build(),
+      // TestNG
+      MethodMatchers.create().ofTypes(ORG_TESTNG_ASSERT)
+        .names(ASSERT_EQUALS, ASSERT_NOT_EQUALS, ASSERT_SAME, ASSERT_NOT_SAME)
         .withAnyParameters()
         .build(),
       // JUnit 5
       MethodMatchers.create().ofTypes(ORG_JUNIT5_ASSERTIONS)
-        .names("assertArrayEquals", "assertEquals", "assertIterableEquals", "assertLinesMatch", "assertNotEquals", "assertNotSame", "assertSame")
+        .names("assertArrayEquals", ASSERT_EQUALS, "assertIterableEquals", "assertLinesMatch", ASSERT_NOT_EQUALS, ASSERT_NOT_SAME, ASSERT_SAME)
```

Comment:
Maybe it would be nice if you move all string literals into constants to have the consistency here